src/components/dialog-full-screen/dialog-full-screen.js

```diff
@@ -24,13 +26,34 @@ import Modal from './../modal';
  * @constructor
  */
 class DialogFullScreen extends Modal {
-
   static defaultProps = {
     open: false,
     enableBackgroundUI: true
   }
 
   /**
+   * A lifecycle method that is called after render.
+   *
+   * @method componentDidMount
+   */
+  componentDidMount() {
```

Comment:
Do we need an 
`componentWillUnmount()`

If the component was somehow unmounted whilst being open. Perhaps this is not possible